benchmarks/abm-attack-fraction/README.md

```diff
@@ -35,13 +36,16 @@ output | [1] | Log posterior density
 Feature | Supported
 ---|---
 Evaluate | True
-Gradient | False
+Gradient | True (via finite difference)
```

Comment:
Since your models don't seem to implement gradients themselves, this should be False. (The via finite difference note in some other benchmarks indicates how the derivatives are computed, but the benchmarks do the actual computation for the user)

Reply:
Also, could you add features and config to the forward model as well? Somewhat redundant here, but better for completeness